### ISSUE TITLE
check config from UI not working properly for duplicated component

### ISSUE THREAD

**Issue by CiquattroFPV** [home-assistant/hassio#739]
I have added a double component on configuration.yaml

```
geo_location:
  - platform: geo_json_events
    url: https://raw.githubusercontent.com/emergenzeHack/terremotocentro/master/_data/issues.geojson
    radius: 250
    entity_namespace: 'Terremoto Centro'

geo_location:
  - platform: geo_json_events
    url: https://earthquake.usgs.gov/earthquakes/feed/v1.0/summary/all_day.geojson
    radius: 250
    entity_namespace: 'USGS_Earthquakes'    
```

when I have done the check config from User Interface it says was okay, but when I have done the check from SSH it says that was an error:

Hassio System log:
```

18-10-05 14:38:01 INFO (MainThread) [hassio.homeassistant] Updated Home Assistant API token
18-10-05 15:13:01 INFO (MainThread) [hassio.homeassistant] Updated Home Assistant API token
18-10-05 15:19:05 INFO (MainThread) [hassio.api.security] /homeassistant/check access from core_ssh
18-10-05 15:19:05 INFO (SyncWorker_10) [hassio.docker] Run command 'python3 -m homeassistant -c /config --script check_config' on homeassistant/raspberrypi3-64-homeassistant
```

SSH Check Config:
```

core-ssh:~# hassio ha check
ERROR
INFO:homeassistant.util.package:Attempting install of colorlog==3.1.4
Testing configuration at /config
ERROR:homeassistant.util.yaml:YAML file /config/configuration.yaml contains duplicate key "geo_location". Check lines 271 and 277.
```

**Comment by cogneato** [comment]
I can confirm that when creating a duplicate of a component, the UI check comes back as valid, but the check config script finds it to be an error.

**Comment by pvizeli** [comment]
They use simple the same API, but will check it

**Comment by pvizeli** [comment]
I close this because I fix the bug in 0.80.0 on home assistant
